disperser/cmd/dataapi/main.go

```diff
@@ -108,6 +120,7 @@ func RunDataApi(ctx *cli.Context) error {
 				DisperserHostname:  config.DisperserHostname,
 				ChurnerHostname:    config.ChurnerHostname,
 				BatcherHealthEndpt: config.BatcherHealthEndpt,
+				QuorumIds:          quorumIds,
```

Comment:
It seems not natural to have this as part of dataapi config, compared to e.g. having the quorum count fetch inside the NewOperatorHandler, which makes code change more contained.

Reply:
NewOperatorHandler does not return err, so I moved the quorum ID query into main. I can refactor to keep it more contained.

Reply:
Updated. PTAL